app/Http/Controllers/Bible/LibraryController.php

```diff
@@ -527,6 +530,10 @@ public function volume()
 
         $this->getBiblesByFilesetId($filesets);
 
+        if ($dam_id) {
```

Comment:
It had been removed because the main query does the search using the next conditions:

```php
// id = dam_id
$query->where('bible_filesets.id', $id)
      ->orWhere('bible_filesets.id', substr($id, 0, -4))
      ->orWhere('bible_filesets.id', 'like',  substr($id, 0, 6))
      ->orWhere('bible_filesets.id', 'like', substr($id, 0, -2) .
```
BUT, it has been added again because some test expected a only one result by dam_id but it can return more than one result according to the above query.